NineChronicles.Headless/GraphTypes/States/Models/Item/EquipmentType.cs

```diff
@@ -13,7 +13,8 @@ public EquipmentType()
             Field<NonNullGraphType<GuidGraphType>>(nameof(Equipment.ItemId));
             Field<NonNullGraphType<IntGraphType>>(nameof(Equipment.level),
                 resolve: context => context.Source.level);
-            Field<NonNullGraphType<IntGraphType>>(nameof(Equipment.Exp));
+            Field<NonNullGraphType<IntGraphType>>(nameof(Equipment.Exp),
```

Comment:
Not related to this pull request but the `Equipment.Exp` seems like `long` type.

https://github.com/planetarium/lib9c/blob/d55cbce0706e8443986cdd58b0e2412b17f3def1/Lib9c/Model/Item/Equipment.cs#L21